src/encoded/static/components/forms/UserRegistrationForm.js

```diff
@@ -183,9 +183,10 @@ export default class UserRegistrationForm extends React.PureComponent {
             ajax.load(
                 endpoint,
                 (resp) => {
+                    const jwtToken = resp.getHeader('X-Request-JWT');
                     // TODO
                     this.setState({ 'registrationStatus' : 'success-loading' });
-                    onComplete(); // <- Do request to login, then hide/unmount this component.
+                    onComplete(jwtToken); // <- Do request to login, then hide/unmount this component.
```

Comment:
This edit isn't necessary. `onComplete`/`onRegistrationComplete` is already bound to `this` & `idToken`, so don't need to pass anything else into it. [Checkout bind at MDN.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Function/bind) (`bind` clones a function and sets the `this` of the newly-cloned function to first parameter of the `bind` call, and sets subsequent parameters of the `bind` call as params always passed into the cloned function).

Reply:
Kk, reverted the commit.